app/models/api/openid_connect/scope.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module OpenidConnect
+    class Scope < ActiveRecord::Base
+      has_many :authorizations, through: :authorization_scopes
+
+      validates :name, presence: true, uniqueness: true
+    end
+  end
+end
```

Comment:
I wonder if it's really worth to have this as a model, or if a constant of valid values and a serialized field on the authorization model wouldn't suffice.